src/System-Support/SystemDictionary.class.st

```diff
@@ -405,6 +415,11 @@ SystemDictionary >> renameClassNamed: oldName as: newName [
 	oldClass rename: newName
 ]
 
+{ #category : #removing }
+SystemDictionary >> reservedVariables [
+	^vars ifNil: [ vars := ReservedVariable lookupDictionary]
```

Comment:
what is the reason to not name it #reservedVariable ?

Reply:
No reason, should be done

Reply:
(I was thinking of using the term "PseudoVariable" instead of Reserved, but that is independed of this change)

Reply:
Actually I also consider PseudoVariable name after my first changes on Variable hierarchy. But I can't recall what stopped me from using it. 
Now pseudo sounds better than reserved in the way what we model with it.

Reply:
But for the name of container variable I would choose the name #reservedVariables because it sounds more general: pseudo vars are reserved by default but we may have others non pseudo but reserved vars.

Having it written I am starting recall my thoughts about these names. What does pseudo actually mean? I think many classes in Variable hierarchy deserves this status preudo. Imagine very special slots. Pseudo feels like characteristics of very special implementation. From the other side it does not matter how variable is implemented to be reserved.

Maybe all of it is just a demagogy :)